src/content/__mocks__/worker-factory.js

```diff
@@ -0,0 +1,21 @@
+import Worker from 'workerjs';
+
+const workerFiles = {
+  'zee-worker': '../../res/zee-worker.js',
+  'worker': '../../dist/worker.js',
+};
+
+const workerInstances = [];
+
+export default class {
+  constructor(file) {
+    const worker = new Worker(workerFiles[file], true);
```

Comment:
in the future I'd like to have a fake worker without a child process, using the `vm` node module. Not sure this is worth the work though.